mrp_production_batch/views/mrp_production_views.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <menuitem
+        id="menu_mrp_production_batch"
+        name="MRP Batch Production"
+        parent="mrp.menu_mrp_manufacturing"
+        action="mrp_production_batch"
+        sequence="21"
+    />
+    <data>
+        <record id="view_mrp_production_tree_inherit" model="ir.ui.view">
+            <field name="name">mrp.production.tree.inherit</field>
+            <field name="model">mrp.production</field>
+            <field name="inherit_id" ref="mrp.mrp_production_tree_view" />
+            <field name="arch" type="xml">
+                <button name="button_plan" position="after">
+                    <button
+                        name="mrp_production_batch_create_wizard_action"
+                        type="object"
+                        string="Batch Production"
+                        class="oe_highlight"
+                    />
+                </button>
+                <field name="product_id" position="before">
+                    <field name="production_batch_id" />
+                </field>
+            </field>
+        </record>
+        <record id="mrp_production_form_view" model="ir.ui.view">
+            <field name="name">mrp.production.form.inherit</field>
+            <field name="model">mrp.production</field>
+            <field name="inherit_id" ref="mrp.mrp_production_form_view" />
+            <field name="arch" type="xml">
+                <xpath
+                    expr="//page[@name='miscellaneous']/group/group[2]"
```

Comment:
Using an element position could lead to unexpected results if someone inherits the view